mac/hid.c

```diff
@@ -432,6 +433,7 @@ struct hid_device_info  HID_API_EXPORT *hid_enumerate(unsigned short vendor_id,
 		if ((vendor_id == 0x0 || vendor_id == dev_vid) &&
 		    (product_id == 0x0 || product_id == dev_pid)) {
 			struct hid_device_info *tmp;
+			bool is_hid; /* Is this an actual HID usb device */
```

Comment:
Maybe rename this to `is_usb_hid` since technically an IOHIDDeviceRef is a HID device object

Reply:
good idea

Reply:
Have updated it and pushed